modules/web/src/app/node-data/component.ts

```diff
@@ -676,4 +685,10 @@ export class NodeDataComponent extends BaseFormValidator implements OnInit, OnDe
       replicas: this._nodeDataService.nodeData.count,
     };
   }
+
+  private _isClusterAutoscalingAppExist(): boolean {
+    return !!this._applicationService.applicationDefinitions.find(
+      app => app.name === 'CLUSTER_AUTOSCALING_APP_DEF_NAME'
```

Comment:
`CLUSTER_AUTOSCALING_APP_DEF_NAME` is a constant so it should be treated as a string value.
```suggestion
      app => app.name === CLUSTER_AUTOSCALING_APP_DEF_NAME
```